### DISCUSSION TITLE
Drawing using Circuit_macros

### DISCUSSION THREAD

**Discussion by FabriceSalvaire** [xesscorp/skidl#113] [Ideas]
_I am trying Github discussion for the first time ..._

Well I started PySpice with a teaching/learning focus, and drawing is an issue... Kicad output is not so good. I found https://ece.uwaterloo.ca/~aplevich/Circuit_macros/  It is a bit voodoo and I love M4, but it does a good job.

I wonder if we can do something...

**Comment by DwightAplevich** [comment]
Would you like any help?  Voodoo? Yes, probably, and it has evolved over 30 years so some parts show their origins if one knows where to look.  The git repository is https://gitlab.com/aplevich/Circuit_macros, which is synced with the CTAN version.  The version on ece.uwaterloo.ca is the development version and possibly a little less stable..
